torch/_dynamo/variables/torch.py

```diff
@@ -419,6 +419,26 @@ def get_state_from_generator():
             return TorchVariable(torch.add, **options).call_function(
                 tx, [args[0], result], {}
             )
+        elif self.value == torch._C._nn.scaled_dot_product_attention:
```

Comment:
I figured I don't need the same logic for _backward since that is not expose in python